exercises/concept/valentines-day/src/ValentinesDay.hs

```diff
@@ -0,0 +1,21 @@
+module ValentinesDay where
+
+-- Define the remaining data types to implement rateActivity.
+-- You are provided the resulting type (`Approval`) to get you started.
+-- You have to implement the other types (Cuisine, Genre, and Activity),
+-- but they do not need to `derive` from type classes.
+
+data Approval
+  = Yes
+  | No
+  | Maybe
+  deriving (Show, Eq)
+
+-- data Cuisine =
+
+-- data Genre =
+
+-- data Activity =
+
+-- rateActivity :: Activity -> Approval
```

Comment:
Why not have this be uncommented? There will be compile errors anyway 🤷

Reply:
Good point. I am changing this pending a way forward with out deriving  type classes.